MAUI/Scheduler/migration.md

```diff
@@ -2454,6 +2470,104 @@ this.Content = scheduler;
 
 </table> 
 
+### DragDropSettings
```

Comment:
why is this in technical term? **DragDropSettings**

Reply:
Class name table mentioned in migration table.